sympy/core/power.py

```diff
@@ -124,6 +124,15 @@ class Pow(Expr):
     | (-oo)**oo    | nan     |                                               |
     | (-oo)**-oo   |         |                                               |
     +--------------+---------+-----------------------------------------------+
+    | oo**I        | nan     |                                               |
+    | (-oo)**I     |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(1+I)    | zoo     |                                               |
+    | (-oo)**(1+I) |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(-1+I)   | 0       |                                               |
+    | (-oo)**(-1+I)|         |                                               |
```

Comment:
> Going by this
<https://github.com/gxyd/sympy/blob/inf_run_time/sympy/core/power.py#L83> line,
I think what you are doing is Ok.
Yes.
Probably i should include `I`(imaginary unit) in there as well?


On Sat, Oct 24, 2015 at 7:50 PM, Sartaj Singh <notifications@github.com>
wrote:

> In sympy/core/power.py
> <https://github.com/sympy/sympy/pull/10029#discussion_r42934384>:
>
> > @@ -124,6 +124,15 @@ class Pow(Expr):
> >      | (-oo)**oo    | nan     |                                               |
> >      | (-oo)**-oo   |         |                                               |
> >      +--------------+---------+-----------------------------------------------+
> > +    | oo**I        | nan     |                                               |
> > +    | (-oo)**I     |         |                                               |
> > +    +------------------------------------------------------------------------+
> > +    | oo**(1+I)    | zoo     |                                               |
> > +    | (-oo)**(1+I) |         |                                               |
> > +    +------------------------------------------------------------------------+
> > +    | oo**(-1+I)   | 0       |                                               |
> > +    | (-oo)**(-1+I)|         |                                               |
>
> Going by this
> <https://github.com/gxyd/sympy/blob/inf_run_time/sympy/core/power.py#L83>
> line, I think what you are doing is Ok.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/10029/files#r42934384>.
>